### PR TITLE
OBJ Parser

### PR DESCRIPTION
**Pull request by toddgr**

### CODE REVIEW

**Comment by Exiled1** [comment]
Is there any reason that theres two obj loader files?

File: Cargo.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ members = [
     "example_plugins/gamepad",
     "example_plugins/keyboard",
     "example_plugins/chat",
+    "example_plugins/obj_parser"
```
Comment (Masterchef365) [inline Cargo.toml:21]: This should now just be `"obj_parser"`, since it was moved to the crate root.

File: obj_parser/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+cimvr_common = { path = "../../common" }
```
Comment (Masterchef365) [inline obj_parser/Cargo.toml:12]: These two deps should be updated to reflect the new location of the crate! See `cargo check`.

File: obj_parser/src/obj.rs
[the remark this answers is not in the document]
Comment (Masterchef365) [reply obj_parser/src/obj.rs:1]: (Resolved)

**Review by Masterchef365 — CHANGES_REQUESTED** [review]
This is looking good! I'd like to suggest one change, which is to make the obj parser into a library instead of a plugin and a library like it is now. Then we can add an example plugin which uses the library. This way, other plugins such as the demo room can import and use the obj parser. The example plugin would use the library in the same way as other plugins would, so that it's clear how it's supposed to be imported and used.

**Comment by Exiled1** [comment]
Plz update the obj loader with main.

**Pull request closed without merging**